cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -571,7 +571,7 @@ def test_custom_button_role_access_service(
 
 @test_requirements.customer_stories
 @pytest.mark.meta(automates=[BZ(1439883)])
-@pytest.mark.provider([VMwareProvider], override=True)
+@pytest.mark.provider([VMwareProvider], override=True, selector=ONE)
```

Comment:
There is no need for `override=True` anymore.

Reply:
Yup, pytest-5 but my intention was toa restrict with ONE of vsphere. Adding :+1: